apps/foundation-demo/src/app/thank-you/components/print-order-summary/print-order-summary.component.html

```diff
@@ -0,0 +1 @@
+<button type="button" class="button large expanded" (click)="printOrderSummary()">Print Order Summary</button>
```

Comment:
Should use `daff-button`

Reply:
Elain actually changed this in https://github.com/graycoreio/daffodil/pull/186